### PR TITLE
feat: Add `NumberControl` and `NumericControl` tests

### PR DESCRIPTION
**Pull request by NathanFarmer**
[Corresponding functional changes](https://github.com/great-expectations/jsonforms-antd-renderers/pull/16) are in base branch

Add tests for `NumberControl` and related components/functions.

### CODE REVIEW

File: tsconfig.production.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
         "src"
     ],
     "exclude": [
-        "**/*.test.tsx",
```
Comment (NathanFarmer) [inline tsconfig.production.json:6]: It was attempting to bundle the `.ts` test file

File: src/jsonforms-test-utils.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  }
+
+type RenderProps<T> = {
+  schema: JSONSchema
+  data?: T
+  uischema?: UISchema
+  onChange?: (result: { data: T }) => void
+}
+
+export function render<T>({ schema, data, uischema, onChange }: RenderProps<T>): ReturnType<typeof RTLrender>{
```
Comment (DrewHoo) [inline src/jsonforms-test-utils.tsx:26]: Is this moved? I thought this already existed in the repo?

Comment (NathanFarmer) [reply src/jsonforms-test-utils.tsx:26]: No, I didn't see it anywhere. I think I found it now though...

Comment (NathanFarmer) [reply src/jsonforms-test-utils.tsx]: Used the existing function, but had to make some tweaks.

**Comment by NathanFarmer** [comment]
> I'm wondering if we want to put all the test code in a test dir so it's easier not to bundle them within the build? Or perhaps that doesn't matter?

@TrangPham I don't think the bundling thing is much of a concern after the `tsconfig` change in this PR. Personally, I prefer to have a separate tests directory just for readability.

**Pull request merged**